hops/numpy_helper.py

```diff
@@ -36,9 +125,65 @@ def save(hdfs_filename, data):
     """
     local_file = os.path.basename(hdfs_filename)
     np.save(local_file, data)
+    _copyToHdfsOverwrite(hdfs_filename)
+
+def savez(hdfs_filename, *args, **kwds):
+    """
+    Save several arrays into a single file in uncompressed .npz format in HDFS
+    If arguments are passed in with no keywords, the corresponding variable names, in the .npz file, are ‘arr_0’, ‘arr_1’, etc. 
+    If keyword arguments are given, the corresponding variable names, in the .npz file will match the keyword names.
+
+    Args:
+       :hdfs_filename: You can specify either a full hdfs pathname or a relative one (relative to your Project's path in HDFS)
+       :args : Arguments, optional
```

Comment:
Might break the docs otherwise
```suggestion
       :args: Arguments, optional
```